doc/content/_index.md

```diff
@@ -2,7 +2,12 @@
 title: "Overview"
 description: ""
 weight: 1
-menu:
-  main:
-    weight: 1
 ---
+
+Welcome to the documentation of TTN Stack!
+
+Here you find a getting started and other guides for common scenarios, as well as learn about concepts and components.
```

Comment:
"here you can find [...] as well as learn" sounds a bit weird, but we can fix that later